anaconda_project/internal/conda_api.py

```diff
@@ -21,6 +21,8 @@
 from anaconda_project.internal.directory_contains import subdirectory_relative_to_directory
 from anaconda_project.internal.py2_compat import is_string
 
+CONDA_EXE = os.environ.get("CONDA_EXE", "conda")
```

Comment:
This is necessary for versions of conda that do not put conda on the path by default, choosing instead to use a shell script.